samples/ComputeSharp.SwapChain/Shaders/HelloWorld.cs

```diff
@@ -1,28 +1,27 @@
-﻿namespace ComputeSharp.SwapChain.Shaders
+﻿namespace ComputeSharp.SwapChain.Shaders;
```

Comment:
Please drop this commit as well, as it's unrelated from this PR.
Also this missed the compute shader copy of this shader anyway. Let's just do both in a separate PR 🙂

Reply:
Done